EXT_mesh_features/ExplicitFeatureIdsAndSimpleProperty/README.md

```diff
@@ -0,0 +1,50 @@
+# Explicit Feature IDs and Simple Property
+
+This sample demonstrates usage of the [`EXT_mesh_features`](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features) extension for storing feature IDs and properties associated with vertices of a mesh.
+
+### Feature IDs
+
+The basic structure and definition of the feature IDs for this example is the same as for the [ExplicitFeatureIds](../ExplicitFeatureIds#feature-ids) example.
+
+Additionally, this sample defines metadata that is associated with the feature IDs. The metadata in this example is the same as in the [ImplicitFeatureIdsAndSimpleProperty](../ImplicitFeatureIdsAndSimpleProperty/) example.
+
+### Metadata Structure
+
+The structure of the metadata is defined with an [`EXT_mesh_features` schema](https://github.com/CesiumGS/glTF/tree/3d-tiles-next/extensions/2.0/Vendor/EXT_mesh_features#schema-definitions) that contains a single class. The class only contains a single property, called `example_VEC3_FLOAT32`. The type of this property is a 3D vector with 32 bit floating-point components, as indicated by the type `"VEC3"` and the component type `"FLOAT32"`. 
+
+### Metadata Instances
```

Comment:
I believe we are calling these "entities", though I sometimes find myself gravitating towards "instances"